packages/plugin-auto-tracking-browser/src/auto-tracking-plugin.ts

```diff
@@ -62,9 +83,11 @@ export const autoTrackingPlugin = (options: Options = {}): BrowserEnrichmentPlug
           return false;
       }
     }
-    const tag = element.tagName.toLowerCase();
+    /* istanbul ignore next */
+    const tag = element?.tagName?.toLowerCase?.();
     /* istanbul ignore if */
-    if (!tagAllowlist.includes(tag)) {
+    if (!DEFAULT_TAG_ALLOWLIST.includes(tag) || !tagAllowlist.includes(tag)) {
+      // Tag needs to be in the default allowlist and the user provided allowlist.
       return false;
     }
     if (cssSelectorAllowlist) {
```

Comment:
As discussed in the last meeting, can we update this to make it easier to allow a single new element without needing to change `tagAllowList`? 

E.g. `cssSelectorAllowlist = ['amp-allow']` would allow tracking this div and button without other configuration changes.

```
<body>
  <button>Click Me</button>
  <div amp-allow>Click me too<./div>
</body>
```

Reply:
Thank you @justin-fiedler for discussing this offline. Updates:
- Remove `tagAllowList` for now, if needed, we can add in the future, addition is easier than deprecation.
- Will track the 6 tags & `cssSelectorAllowlist`.